docs/index.rst

```diff
@@ -53,6 +53,7 @@ If you are interested to contribute, please read `contribution-guide`_ first.
    
    api/rods
    api/rigidbody
+   api/damping
```

Comment:
`damping` below `time_steppers` maybe?

Reply:
The reason for that is the Damping wrapper actually holds a list of the "dampers" initialized through the "_damper" class. Hence the naming. I'm not too stuck up on this and arman can change this if he feels like.